assets/api-swagger/swagger.json

```diff
@@ -105,6 +105,15 @@
                 "false"
               ]
             }
+          },
+          {
+            "name": "includeVariants",
+            "in": "query",
+            "description": "Product variants (or child products) for shippable SKUs.",
+            "required": true,
```

Comment:
Just to confirm if this one is really required.

Reply:
your right, fixed